pkg/cmd/ssh/options.go

```diff
@@ -0,0 +1,228 @@
+/*
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package ssh
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/gardener/gardenctl-v2/internal/util"
+	"github.com/gardener/gardenctl-v2/pkg/cmd/base"
+	"github.com/gardener/gardener/pkg/utils"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh"
+	cryptossh "golang.org/x/crypto/ssh"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+// Options is a struct to support target command
+type Options struct {
+	base.Options
+
+	// Interactive can be used to toggle between gardenctl just
+	// providing the bastion host while keeping it alive (non-interactive),
+	// or gardenctl opening the SSH connection itself (interactive). For
+	// interactive mode, a NodeName must be specified as well.
+	Interactive bool
+
+	// NodeName is the name of the Shoot cluster node that the user wants to
+	// connect to. If this is left empty, grdenctl will only establish the
+	// bastion host, but leave it up to the user to SSH themselves.
+	NodeName string
+
+	// CIDRs is a list of IP address ranges to allowed for accessing the
```

Comment:
>CIDRs is a list of IP address ranges to allowed for accessing the created Bastion host

the "to" does not fit into the sentence or something is missing?
`CIDRs is a list of IP address ranges allowed for accessing the created Bastion host` or
`CIDRs is a list of IP address ranges to be / that should be allowed for accessing the created Bastion host`